File: web/src/app/models/office.ts
import {Address} from './address';
import {Appointment} from './appointment';
import {OfficeHour} from './office-hour';
import {MedicalWorker} from './medicalworker';

export class Office {
  id: number;
  name: string;
  phone: string;
  fax: string;
  address: Address;
  email: string;
  officehours: Array<OfficeHour>;
  appointments: Array<Appointment>;
  officeWorkers: Array<MedicalWorker>;
}

File: mobile/app/src/main/java/wse18/ase/qse03/mobile/model/Office.kt
package wse18.ase.qse03.mobile.model

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import java.io.Serializable

@JsonIgnoreProperties(ignoreUnknown = true)
class Office(
    var id: Long,
    var name: String,
    var phone: String,
    var fax: String,
    var address: Address,
    var email: String,
    var officeWorkers: List<MedicalWorker>,
    var officehours: List<OfficeHour>
) : Serializable {
    constructor() : this(-1, "", "", "", Address(), "", emptyList<MedicalWorker>(), emptyList<OfficeHour>())
}